### ISSUE TITLE
View action links (edit|delete|duplicate entry) disappear when using the {get} merge tag as the "search_value" parameter for the  [gravityview] shortcode

### ISSUE THREAD

**Issue by bennemann** [GravityKit/GravityView#1537]
Here's a regular View with a field and an Edit Entry link:

![ksnip_20201127-172921](https://user-images.githubusercontent.com/5726194/100469559-b3a4f900-30d6-11eb-8399-883a2b87d3d4.png)

It works fine when simply embedded: 

![ksnip_20201127-173250](https://user-images.githubusercontent.com/5726194/100469556-b273cc00-30d6-11eb-89b8-35d10aaeab9d.png)
![ksnip_20201127-173327](https://user-images.githubusercontent.com/5726194/100469555-b1db3580-30d6-11eb-9e00-5db6de5e9527.png)

However, when using the {get} merge tag as the search_value parameter, the Edit Entry link disappear:

![ksnip_20201127-173229](https://user-images.githubusercontent.com/5726194/100469557-b30c6280-30d6-11eb-83dc-113bb5f344c1.png)

![ksnip_20201127-173147](https://user-images.githubusercontent.com/5726194/100469558-b30c6280-30d6-11eb-9b62-b7d2307a0c48.png)

When using a simple string as the search_value parameter or using an Advanced Filter to create the exact same condition the Edit Entry link is not affected.

**Comment by bennemann** [comment]
https://secure.helpscout.net/conversation/1350163886/30942

**Comment by rafaehlers** [comment]
https://secure.helpscout.net/conversation/1435857853/32783?folderId=408730

**Comment by zackkatz** [comment]
@rafaehlers Please reproduce!

**Comment by rafaehlers** [comment]
Still happening on 2.14.5

https://user-images.githubusercontent.com/1222472/170090544-d15078ab-73e0-43b5-9750-e15e46eff6c6.mp4

**Comment by rafaehlers** [comment]
Workaround fix by Gennady:

```<?php
/**
 * Add uncached version of GravityView_Edit_Entry::maybe_not_visible
 */
add_action( 'gravityview/field/is_visible', function( $visible, $field, $view ) {
	remove_action( 'gravityview/field/is_visible', [ GravityView_Edit_Entry::getInstance(), 'maybe_not_visible' ], 10, 3 );
	add_action( 'gravityview/field/is_visible', function( $visible, $field, $view ) {
		if ( 'edit_link' !== $field->ID ) {
			return $visible;
		}

		if ( ! $view instanceof \GV\View ) {
			return $visible;
		}

		foreach ( $view->get_entries()->all() as $entry ) {
			if ( GravityView_Edit_Entry::check_user_cap_edit_entry( $entry->as_entry(), $view ) ) {
				return true;
			}
		}
		return false;
	}, 10, 3 );
	return $visible;
}, 9, 3 );
```

**Comment by rafaehlers** [comment]
https://secure.helpscout.net/conversation/2109852808/46016

**Comment by zackkatz** [comment]
@rafaehlers Please test again!

**Comment by rafaehlers** [comment]
Fixed on 2.17.4 ✅
